src/link.d

```diff
@@ -806,45 +807,49 @@ version (Windows)
         }
         // Normalize executable path separators, see Bugzilla 9330
         cmd = toWinPath(cmd);
-        version (CRuntime_Microsoft)
```

Comment:
So that there is one common code path for all cases and not two code paths depending on the cruntime used. The CreateProcess version will work with both c-runtimes the spawnlp version will not.